plugin.rb

```diff
@@ -31,6 +31,7 @@ module ::DiscoursePrometheus; end
 
 GlobalSetting.add_default :prometheus_collector_port, 9405
 GlobalSetting.add_default :prometheus_trusted_ip_allowlist_regex, ''
+DiscoursePluginRegistry.define_filtered_register :custom_metrics
```

Comment:
Perhaps `DiscoursePluginRegistry.global_collectors` instead since we're specifically adding to the global reporter.

Reply:
Our README is very bare here but this is something that is worth documenting :)